src/daemon/rpmostreed-transaction-types.c

```diff
@@ -191,13 +177,15 @@ package_diff_transaction_execute (RpmostreedTransaction *transaction,
 
   sysroot = rpmostreed_transaction_get_sysroot (transaction);
   upgrader = rpmostree_sysroot_upgrader_new (sysroot,
-					     self->osname,
-					     upgrader_flags,
-					     cancellable,
-					     error);
+                                             self->osname,
+                                             upgrader_flags,
+                                             cancellable,
+                                             error);
   if (upgrader == NULL)
     goto out;
 
+  origin = rpmostree_sysroot_upgrader_get_origin (upgrader);
```

Comment:
Basically I was originally trying to have the origin object be immutable; making it mutable is OK, and definitely allows us to encapsulate things more in the end.  But if we're going to change to mutable let's do it consistently.